cli/tools/jupyter/mod.rs

```diff
@@ -118,9 +118,7 @@ pub async fn kernel(
     fn write(&mut self, buf: &[u8]) -> std::io::Result<usize> {
       self
         .0
-        .send(StreamContent::stdout(
-          String::from_utf8_lossy(buf).into_owned(),
-        ))
+        .send(StreamContent::stdout(&String::from_utf8_lossy(buf)))
```

Comment:
I see why you were hoping to pass in a `String` here. Lossy creates an owned string whereas `std::str::from_utf8` returns a `Result<&str>`. We could have `runtimelib`'s `StreamContent` have a method to accept from `&[u8]` and create the owned string directly. Alternatively, this could just be:


```rust
        StreamContent {
            name: Stdio::Stdout,
            text: String::from_utf8_lossy(buf),
        }
```